composer.json

```diff
@@ -20,6 +20,7 @@
         "php-http/discovery": "~1.2",
         "php-http/httplug": "~1.1",
         "psr/http-message-implementation": "~1.0",
+        "ramsey/uuid": "~3.6",
```

Comment:
Can we lower this to 3.3.0? Just for more BC, since that release is the oldest that I would advise to use (due to the fix to the collision bug).